common/scylla_create_devices

```diff
@@ -38,16 +38,19 @@ class NoDiskFoundError(Exception):
     def __str__(self):
         return "No data disk found, abort setup"
 
-def create_raid(devices):
+def create_raid(devices, raid_level):
     if len(devices) == 0:
         raise(NoDiskFoundError)
+    if raid_level not in [0, 5]:
```

Comment:
the check is actually done in the argument parser, no need to double-check it

Reply:
also set default there

Reply:
fixed